pkg/kubelet/kubelet_pods.go

```diff
@@ -1388,10 +1388,6 @@ func (kl *Kubelet) convertStatusToAPIStatus(pod *v1.Pod, podStatus *kubecontaine
 			IP: ip,
 		})
 	}
-
-	if len(apiPodStatus.PodIPs) > 0 {
```

Comment:
I´m not familiar with this code but ... is this change related to this commit and kubenet?

Reply:
Unsure... I'm suspicious that it's an accident from a rebase, because I definitely didn't add it, and I can't find it in the parent branch https://github.com/kubernetes/kubernetes/pull/79386/files